bridges/snowbridge/primitives/outbound-queue/src/v2/converter/tests.rs

```diff
@@ -670,22 +670,22 @@ fn xcm_converter_with_different_fee_asset_succeed() {
 }
 
 #[test]
-fn xcm_converter_with_fees_greater_than_reserve_succeed() {
+fn xcm_converter_with_fees_greater_than_reserve_fails() {
 	let network = BridgedNetwork::get();
 
 	let token_address: [u8; 20] = hex!("1000000000000000000000000000000000000000");
 	let beneficiary_address: [u8; 20] = hex!("2000000000000000000000000000000000000000");
 
 	let asset_location: Location = [AccountKey20 { network: None, key: token_address }].into();
-	let fee_asset: Asset = Asset { id: AssetId(Here.into()), fun: Fungible(1000) }.into();
+	let fee_asset: Asset = Asset { id: AssetId([GlobalConsensus(NetworkId::Polkadot)].into()), fun: Fungible(2000) }.into(); // Fee is greater than asset amount
```

Comment:
I thought the fee asset would be ether in this context, so I don't understand the change.

In the xcm below, we are withdrawing ether but paying fees in dot which has to come somewhere

Reply:
@yrong ^ can you review this please

Reply:
Yeah, this is not correct. The fee asset should be ether with format as `Here` after reanchored.

But it did reminds me of another potential fix. 

@claravanstaden  You can leave it to me.